posthog/batch_exports/http.py

```diff
@@ -342,9 +328,6 @@ class BatchExportViewSet(TeamAndOrgViewSetMixin, viewsets.ModelViewSet):
     serializer_class = BatchExportSerializer
 
     def get_queryset(self):
-        if not isinstance(self.request.user, User):
```

Comment:
We don't need to check authentication - thats what the TeamAndOrgViewSetMixin is responsible for.